src/main/java/seedu/address/logic/commands/list/ListApplicationCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.commands.list;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_LIST_SUCCESS;
+import static seedu.address.logic.commands.util.CommandUtil.getCommandResult;
+import static seedu.address.model.FilterableItemList.PREDICATE_SHOW_ALL_ITEMS;
+import static seedu.address.model.util.ItemUtil.APPLICATION_NAME;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.ui.tabs.TabName;
+
+/**
+ * todo javadocs
+ */
+public class ListApplicationCommand extends ListCommand {
+
+    public static final String MESSAGE_SUCCESS = String.format(MESSAGE_LIST_SUCCESS, APPLICATION_NAME);
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.getApplicationList().updateFilteredItemList(PREDICATE_SHOW_ALL_ITEMS);
```

Comment:
Should we be using `model.updateFilteredApplicationList()` instead to avoid LoD violations?

Reply:
Thanks for the spot